contracts/extensions/VotingReputation.sol

```diff
@@ -756,21 +756,30 @@ contract VotingReputation is ColonyExtension, PatriciaTreeProofs {
     uint256 stakeFraction = wdiv(stakes[_motionId][_staker][_vote], totalSideStake);
 
     uint256 realStake = wmul(stakeFraction, motion.stakes[_vote]);
-    uint256 requiredStake = getRequiredStake(_motionId);
 
     uint256 stakerReward;
     uint256 repPenalty;
 
     // Went to a vote, use vote to determine reward or penalty
     if (add(motion.votes[NAY], motion.votes[YAY]) > 0) {
 
-      uint256 loserStake = sub(requiredStake, motion.paidVoterComp);
+      uint256 loserStake;
+      uint256 winnerStake;
+      if (motion.votes[YAY] > motion.votes[NAY]){
```

Comment:
Too much?

```
uint256 loserStake = motion.stakes[(motion.votes[YAY] > motion.votes[NAY]) ? YAY : NAY]; 
uint256 winnerStake = motion.stakes[(motion.votes[YAY] > motion.votes[NAY]) ? NAY : YAY]; 
```

Reply:
Yeah, I think so. It's harder to read, and I think it uses more gas as well.